chore: add example for exception and accepted (#43)
Zilliqa/apidocs: source/index.html.md

}
```

> *Note: If the transaction had an data field or code field or exceptions field, it will be displayed*

Reviewer: ```suggestion
*Note: If the transaction had data, code, or exceptions fields, those fields will be displayed*
```
